certbot-ci/certbot_integration_tests/utils/acme_server.py

```diff
@@ -195,3 +196,36 @@ def _launch_command(command, cwd=os.getcwd()):
     except subprocess.CalledProcessError as e:
         sys.stderr.write(e.output)
         raise
+
+
+def main():
+    args = sys.argv[1:]
+    server_type = args[0] if args else 'pebble'
+    possible_values = ('pebble', 'boulder-v1', 'boulder-v2')
+    if server_type not in possible_values:
+        raise ValueError('Invalid server value {0}, should be one of {1}'
+                         .format(server_type, possible_values))
+
+    acme_xdist, start, stop = setup_acme_server(server_type, [], False)
+    process = None
+
+    try:
+        start()
+        print('--> Instance of {0} is running, directory URL is {0}'
+              .format(acme_xdist['directory_url']))
+        print('--> Press CTRL+C to stop the ACME server.')
+
+        docker_name = 'pebble_pebble_1' if 'pebble' in server_type else 'boulder_boulder_1'
+        process = subprocess.Popen(['docker', 'logs', '-f', docker_name])
```

Comment:
Both approaches are very useful. Getting used to the Docker way, I appreciate a lot processes than can be run in foreground. What about proposing a flag to run either in foreground or in background?

Reply:
I'd personally rather us just implement one way to do this now and implement another behind a flag if the demand/need really arises. I like having it run the server in the background because I think it's more flexible (you can run `docker logs` yourself if you want) and it's been years since I had to look at server logs at all. Why do you like running it in the foreground?

Reply:
Well, the last several times I needed to code on acme, it was very useful to have the log in parallel, and see how the server reacted to the modifications.

In general, I prefer to run in foreground because it is quite useful to get a blocking point in a script, as I said for Docker a process needs to be in foreground, and because in shell, it is easier to transform a foreground process into background one than the contrary.

For the specific case of Boulder and Pebble in fact, there is more than on Docker that is started, and also a dedicated network. Shutting down for Boulder requires at least 4 Docker commands. It is easier and less error prone to see it properly cleaned up upon SIGTERM on the foreground command, than to run the Docker command or have a separate dedicated command provided by certbot-ci to do it.

Reply:
> In general, I prefer to run in foreground because it is quite useful to get a blocking point in a script, as I said for Docker a process needs to be in foreground, and because in shell, it is easier to transform a foreground process into background one than the contrary.

I buy this argument. Let's stick with it in the foreground.

Reply:
Acknowleded